base/card-api.gts

```diff
@@ -609,17 +610,27 @@ export class Card {
       }
       return Object.fromEntries(
         Object.entries(getFields(value, { includeComputeds: true }))
-          .map(([fieldName, field]) => [fieldName, getQueryableValue(field!.card, value[fieldName])])
+          .map(([fieldName, field]) => {
+            try {
+              return [fieldName, getQueryableValue(field!.card, value[fieldName])];
```

Comment:
let's not throw not loaded errors here. instead we should check first if the value is missing